RELEASE_NOTES.md

```diff
@@ -18,7 +18,28 @@ CSM 1.2 contains approximately 2000 changes spanning bug fixes, new feature deve
 * Release Cray Automated Network Utility - CANU V1.0.0
 * Performance improvements to Unbound and DHCP Helper
 * Initial Release of Bifurcated CAN
-* PowerDNS including PowerDNS Manager and Troubleshooting Guide
+
+  The user and administrative traffic segregation introduced by Bifurcated CAN has changed the URLs for certain services as it is now necessary to include the network path in the fully qualified domain name.
+
+  The following table assumes the system was configured with a `system-name` of `shasta` and a `site-domain` of `dev.cray.com`.
+
+  | Old Name                         | New Name                                |
+  |----------------------------------|-----------------------------------------|
+  | auth.shasta.dev.cray.com         | auth.cmn.shasta.dev.cray.com            |
+  | nexus.shasta.dev.cray.com        | nexus.cmn.shasta.dev.cray.com           |
+  | grafana.shasta.dev.cray.com      | grafana.cmn.shasta.dev.cray.com         |
+  | prometheus.shasta.dev.cray.com   | prometheus.cmn.shasta.dev.cray.com      |
+  | alertmanager.shasta.dev.cray.com | alertmanager.cmn.shasta.dev.cray.com    |
+  | vcs.shasta.dev.cray.com          | vcs.shasta.dev.cray.com                 |
+  | kiali-istio.shasta.dev.cray.com  | kiali-istio.shasta.dev.cray.com         |
+  | s3.shasta.dev.cray.com           | s3.shasta.dev.cray.com                  |
+  | sma-grafana.shasta.dev.cray.com  | sma-grafana.shasta.dev.cray.com         |
+  | sma-kibana.shasta.dev.cray.com   | sma-kibana.shasta.dev.cray.com          |
```

Comment:
missing the cmn in right column for vcs, kiali-istio, s3, sma-grafana, and sma-kibana.

Reply:
I added the missing `cmn` network path.